CustomApps/lyrics-plus/Settings.js

```diff
@@ -475,7 +475,16 @@ function openConfig() {
 					key: "highlight-color",
 					type: ConfigInput,
 					when: () => !CONFIG.visual["colorful"]
-				}
+				},
+				{
+					desc: "Text convertion: Detection threshold",
+					info: "(Distinguish between Japanese and Chinese; in percentage)",
```

Comment:
Please be more descriptive of the settings option so that new users could understand
Threshold/Percentage of what?

Reply:
I have changed the description as an attempt.

Reply:
Just updated the description again.